app/models/user.rb

```diff
@@ -459,6 +466,14 @@ def set_projects_limit
     self.projects_limit = current_application_settings.default_projects_limit
   end
 
+  def set_current_streak
```

Comment:
We need this to reset current_streak to zero automatically when the last_contributed day is pass by.